include/fast_float/parse_number.h

```diff
@@ -99,7 +99,24 @@ fastfloat_really_inline bool rounds_to_nearest() noexcept {
   //
   // Note: This may fail to be accurate if fast-math has been
   // enabled, as rounding conventions may not apply.
+  #ifdef _MSC_VER
```

Comment:
You may want `#if FASTFLOAT_VISUAL_STUDIO`. Compiling with the clang front-end under Visual Studio (`Clang-cl`) might still leave `_MSC_VER` defined.

See https://github.com/fastfloat/fast_float/blob/main/.github/workflows/vs17-clang-ci.yml